packages/dd-trace/src/plugin_manager.js

```diff
@@ -134,9 +135,12 @@ module.exports = class PluginManager {
     }
 
     // TODO: update so that it's available for every CI Visibility's plugin
-    if (name === 'mocha') {
+    if (name === 'mocha' || name === 'jest') {
       sharedConfig.isAgentlessEnabled = experimental && experimental.exporter === 'datadog'
     }
+    if (name === 'jest') {
```

Comment:
Similar to the above, why isolate the config here? Also, what is this config in the first place?